bin/main.rb

```diff
@@ -1,90 +1,76 @@
 #!/usr/bin/env ruby
-puts 'Wellcome to TicTacToe'
-puts 'Player1 Name'
-player1 = gets.strip
+require_relative '../lib/board'
+require_relative '../lib/players'
+require_relative '../lib/helper'
+require 'colorize'
+require 'tty-font'
+require 'tty-prompt'
 
-puts 'Player2 Name'
-player2 = gets.strip
+font = TTY::Font.new(:standard)
+puts font.write('TicTacToe')
+
+puts 'Welcome To TicTacToe'.upcase.colorize(:cyan)
+puts
+puts 'To play the game, follow instructions below.'
+puts '---------------------------------------'
+puts '|------------INSTRUCTIONS----------------|'
+puts '1. Both players should enter their names.'
+puts '2. A TicToe board will presented to you with numbers 1 through 9.'
+puts "3. Enter numbers 1 through 9 that corresponds to the position on the board that you'd like to occupy."
+puts
+puts 'May the force be with you.'.upcase.colorize(:magenta)
+puts
+
+prompt = TTY::Prompt.new
+
+player1_name = prompt.ask("Player 1, What's your name?")
+player2_name = prompt.ask("Player 2, What's your name?")
 
 system('clear')
 
-board = (1..9).to_a
+turn = 0
 
-def map_to_index(position)
-  position - 1
-end
+# Object Creation.
+player_one = Player.new(player1_name, 'X')
+player_two = Player.new(player2_name, 'O')
+game = GameBoard.new
```

Comment:
The players variables must be attributes of the `GameBoard` class.
If you don't want, you can create a `Game` class. Because without the Game object the players doesn't exists.